custom_components/e3dc_rscp/sensor.py

```diff
@@ -372,6 +372,92 @@
         device_class=SensorDeviceClass.POWER_FACTOR,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    SensorEntityDescription(
```

Comment:
Could you set this as 
`entity_category=EntityCategory.DIAGNOSTIC,`
it'll appear along the other diagnostics information at the bottom of the device page. See at the top of the file for more examples. Sorry, missed that in the review.

Reply:
Good point, yes will do.
(in fact it does only report "0" - i found in a discussions somewhere ( think at python-e3dc) that this seems to be normal)